helm/oaf/values.yaml

```diff
@@ -1,3 +1,28 @@
+mtn_rwanda_connector:
+  enabled: true
+  tenants: "oaf"
+  zeebe_broker_contactpoint: "zeebe-zeebe-gateway:26500"
+  camel_server_port: 5000
+  mtnrw_callback:
+  mtnrw_subscriptionkey:
+  mtnrw_environment:
+  mtnrw_api_host:
+  mtnrw_client_secret:
+  mtnrw_client_key:
+  spring_profiles_active: "bb"
+  mtn_max-retry_count: 3
+  mtn_api_timeout: 6000
+  deployment:
+    resources:
+      limits:
+        memory: 1000m
```

Comment:
These default values are too high. CPU requests are usually 100m and memory is usually 128Mi